### ISSUE TITLE
Chapter 5 Sections 5.3.5, 5.4.1, 5.4.2. Fuzzy on filters (Steve Gillespie)

### ISSUE THREAD

**Issue by sgillesp-ai** [KGConf/Bookclub-DemystifyingOWL#30]
Basic question: Could you please clarify what a **filter class** is? I’ve gathered clues but I am still a little fuzzy. 

Here is what I have so far: 
•	One instance of filter class is when a class expression (defined as a new class) is used in a property restriction. (5.3.5, p 112)
•	The filter is a class that constrains the object of grounding triples (5.4.2, p 128)
•	It can be a class for object restrictions and a datatype for data restrictions (bottom of p 128)
•	It can be a mandatory for universal, existential and individual value restrictions and is optional for cardinality (p 129)

**Comment by uscholdm** [comment]
Good question. I now see that I did not properly introduce the term 'filter class' before I started using it.  The easiest way to get the idea is to look a the 6 examples on p 123.  

1. Existential: The set of all things that cover some party (in the context of insurance).
`(doe:cover some doe:Party)`

1. Universal: The set of all things that have only physicians as members.
`(doe:hasMember only doe:Physician)`

1. Minimum cardinality: The set of all things that are verified by at least 2 people.
`(doe:verifiedBy min 2 doe:Person)`

1. Maximum cardinality: The set of all things that have no more than 3 registered devices.
`(doe:registeredOn max 3 doe:Device)`

1. Exact cardinality: The set of all things that have exactly two wheels.
`(doe:hasPart exactly 2 doe:Wheel)`

1. Individual value: The set of all things that are part of Sentara Healthcare.
`(doe:partOf value :_SentaraHealthcare)`

The filter is at the end of the restriction expressions. In order, the filters are `Party`, `Physician`, `Person`, `Device`, `Wheel` and `_SentaraHealthcare`.  In all but the last case, the filter is a class (hence the term 'filter class'). If datatype properties were used, the filters would be datatypes such as `string` or `integer`.  

In the last case the filter is an individual, but it would be a literal if it was a data property restriction.   _In all cases_, the filter is saying something about the objects of the grounding triples. Here is a quote from the top of p128:

> [The filter on a a restriction is a] qualification that constrains what the type of the objects of the restriction’s grounding triples may or must be.
